### PR TITLE
AEM Migration Phase 2

### PR DESCRIPTION
**Pull request by prycrane**
[AEM Migration Phase 2](https://test.canada.ca/experimental/migration/aem-migration.html)

English pull request: [AEM Migration Phase 2 #253](https://github.com/canada-ca/design-system/pull/253)

____________________________________________________________________________________________________________

### Preview to test for migration (pr#150)
The preview is generated from https://github.com/canada-ca/design-system/pull/150 (Draft). The config file is the same.

**À propos de Canada.ca**
https://deploy-preview-150--systeme-conception-canada-ca.netlify.app/a-propos/

**Bureau de la transformation numérique**
https://deploy-preview-150--systeme-conception-canada-ca.netlify.app/a-propos/bureau-transformation-numerique.html

**Tâches principales pour Canada.ca**
https://deploy-preview-150--systeme-conception-canada-ca.netlify.app/a-propos/taches-principales-pour-canada-ca.html

**Derniers changements apportés au système de conception de Canada.ca**
https://deploy-preview-150--systeme-conception-canada-ca.netlify.app/a-propos/derniers-changements.html

**Guide de rédaction du contenu du site Canada.ca**
https://deploy-preview-150--systeme-conception-canada-ca.netlify.app/guide-redaction/

### CODE REVIEW

File: a-propos/a-propos.md
Comment (delisma) [inline a-propos/a-propos.md:1]: File name should be index.md, bureau-transformation-numerique.md and taches-principales-pour-canada-ca.md. Their urls would then read conception.canada.ca/a-porpos, conception.canada.ca/a-propos/bureau-transformation-numerique.html and conception.canada.ca/a-propos/taches-principales-pour-canada-ca.html. I'm not sure if it's SEO friendlier but it feel more human readable.

Comment (prycrane) [reply a-propos/a-propos.md:1]: 10-4.  You're reading my mind.  I was going to ask you about that.

**Comment by donohuech** [comment]
@prycrane The Guide de rédaction du contenu du site Canada.ca hhttps://deploy-preview-150--systeme-conception-canada-ca.netlify.app/guide-redaction/ seems to have the width constrained on it. The text is only going halfway across the page, whereas on the live version it goes across the whole width of the page (https://www.canada.ca/fr/secretariat-conseil-tresor/services/communications-gouvernementales/guide-redaction-contenu-canada.html)

**Comment by donohuech** [comment]
@prycrane Visited all 5 links to the migrated pages and clicked around other parts of the design system and everything looks good (just the one previous comment about width in Style Guide).

**Pull request closed without merging**